app-modules/interaction/graphql/interaction-campaign.graphql

```diff
@@ -0,0 +1,96 @@
+type InteractionCampaign
+    @model(class: "AdvisingApp\\Interaction\\Models\\InteractionCampaign") {
+    "Unique primary key."
+    id: ID!
+
+    "The name of the interaction campaign."
+    name: String!
+
+    "Interactions related to this interaction campaign."
+    interactions: [Interaction!] @hasMany
+
+    "The created date of the interaction campaign."
+    created_at: DateTime
+
+    "The updated date of the interaction campaign."
+    updated_at: DateTime
+
+    "The deleted date of the interaction campaign."
+    deleted_at: DateTime
+}
+
+input InteractionCampaignQuery {
+    "The unique primary key of the interaction campaign."
+    id: ID
+
+    "The name of the interaction relation."
+    name: String
+
+    "The interactions related to this interaction campaign."
+    interactions: InteractionsQuery
+
+    "The created date of the interaction campaign."
+    created_at: DateTime
+
+    "The updated date of the interaction campaign."
+    updated_at: DateTime
+
+    "The deleted date of the interaction campaign."
+    deleted_at: DateTime
+}
+
+extend type Query {
+    "Get a specific interaction campaign by id."
+    interactionCampaign(id: ID! @eq): InteractionCampaign
+        @find
+        @canResolved(ability: "view")
+
+    "List multiple interactions campaigns."
+    interactionCampaigns(
+        "Filter by the interaction campaigns attributes and relations."
+        where: InteractionCampaignQuery @searchBy
+    ): [InteractionCampaign!]! @paginate @canModel(ability: "viewAny")
```

Comment:
`@softDeletes`

I can also clean these up if you want when I update the alerts api.